sdk/spring/azure-spring-boot-starter-active-directory-b2c/src/main/resources/aadb2c.enable.config

```diff
@@ -0,0 +1 @@
+dummy
```

Comment:
Does this really need to be in here? I'm not going to block approval on the PR but I am curious.

Reply:
Yes, we'll auto-configure the AAD B2C starter when this file is on the classpath.